crates/biome_cli/src/execute/traverse.rs

```diff
@@ -265,9 +265,9 @@ fn traverse_inputs(
                 .map(|path| {
                     if let Some(working_directory) = fs.working_directory() {
                         if path.starts_with(".") || path.starts_with("./") {
-                            working_directory
```

Comment:
That was a mistake of mine. The intention was to use absolute paths when users write `.` or `./`. Instead, we should just check if the path is **equal** to `.`/`./`, instead of using `starts_with`